src/Containers/WeatherToday/weatherToday.tsx

```diff
@@ -0,0 +1,132 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import {WeatherCard} from "./components/WeatherCard/WeatherCard";
+import {SearchBar} from "./components/searchBar/searchBar";
+import {PreloaderComponent} from "../../Component/preloader/preloader";
+
+export const WeatherToday = (): JSX.Element => {
+    const [cityTitleValue, setCityTitleValue] = useState<string>('')
+    const [cityTitle, setCityTitle] = useState<string>('')
```

Comment:
What's the difference between cityTitleValue and cityTitle? It should not be like this.